README.md

```diff
@@ -705,7 +708,7 @@ For example:
 :LualineRenameTab Project_K
 ````
 It's useful when you're using rendering mode 2/3 in tabs.
-To unname a tablage run `:LualineRenameTab` without argument.
+To rename/unname a tabpage run `:LualineRenameTab` without argument.
```

Comment:
To remove name ?

If additional argument isn't provided `:LualineRenameTab` removes the currently set name.

Reply:
Good catch! Fixed